.gitignore

```diff
@@ -28,4 +28,4 @@ openapi.json
 /blob-report/
 /playwright/.cache/
 
-.env
```

Comment:
these two lines are equivalent.  but `**/.env` is a bit more verbose. i would suggest you revert this

Reply:
I believe `**/.env` targets all `.env` files in any directory (including root), whereas `.env` only targets the root's `env`.
Since we have to `.env` files, I thought we may want a rule to to ignore all.

Reply:
A plain filename will match any file that has the same name.  Try it out on your computer.